codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    require_ci_to_pass: no
+
+coverage:
+  status:
+    patch:
+      default:
+        target: 95%
```

Comment:
I think we've set 100% as our passing codecov in other testing configs

Reply:
Yep, will fix it